app/tests/utils.lisp

```diff
@@ -0,0 +1,18 @@
+(in-package :qvm-app-tests)
+
+(defmacro with-mocked-defun (defs &body body)
```

Comment:
@stylewarning having trouble coming up with a simple docstring that will satisfy you

Reply:
Call it `with-mocked-function-definitions` and have a docstring like:

>Temporarily (i.e., dynamically) re-define global function definitions dictated by DEFs over the course of BODY.
>
> DEFS are LET-like bindings (not FLET!) whose binding keys are symbols that name global functions, and whose binding values are evaluated (just as with LET) and return functions. These functions should have conforming lambda lists, types, and return values to the existing global functions.